server/utils/licenceComparator.ts

```diff
@@ -82,68 +103,14 @@ const compareAdditionalConditionSet = (
         })
       }
 
-      originalCondition = sortedOriginalConditionSet.shift()
-      variedCondition = sortedVariedConditionSet.shift()
-    }
-  }
-
-  const newZones = variedExclusionZoneConditionsSet.map(c => c.expandedText).join('\n\n')
-
-  if (originalExclusionsZoneConditionsSet.length === 0 && variedExclusionZoneConditionsSet.length > 0) {
-    variedConditionsBuilder.recordConditionAdded({
-      category: 'Freedom of movement',
-      condition: `${newZones}`,
-    })
-  } else if (changesToMultipleExclusionZones(variedExclusionZoneConditionsSet, originalExclusionsZoneConditionsSet)) {
-    if (newZones.length === 0) {
-      variedConditionsBuilder.recordConditionRemoved({
-        category: 'Freedom of movement',
-        condition: `Exclusion zones have been removed`,
```

Comment:
This used to be a MEZ-specific method. Have confirmed with Tom that it's fine to remove this and instead to just list the text of the removed condition instances